spec/system/application_sitemap_spec.rb

```diff
@@ -9,7 +9,7 @@
       document = Nokogiri::XML::Document.parse(body)
       nodes = document.search("url")
 
-      expect(nodes.count).to eq(292)
+      expect(nodes.count).to eq(294)
```

Comment:
This is a magic number. The latest number was changed when a new role was introduced. I understood that introducing new job roles makes the sitemap to include the extra nodes, and... went with it! 
I personally believe this test probably could be axed, but prefer to kick the can forward as was easy to just increase the number as needed :shrug: